doc/contributing/maintaining-web-assembly.md

```diff
@@ -78,13 +78,6 @@ Node.js GitHub organization. As needed, an updated copy
 is vendored into the Node.js deps in
 [deps/uvwasi](https://github.com/nodejs/node/tree/main/deps/uvwasi).
 
-To update the copy of uvwasi in the Node.js deps:
-
-* Copy over the contents of `include` and `src` to the corresponding
-  directories.
-* Check if any additional files have been added and need to be added
```

Comment:
Is there any way to check this in the update script and to generate a warning/info message if there are new files found?

Reply:
That’s a valid point, maybe @marco-ippolito already faced something alike in other scripts, I'll ask him!

Reply:
this is the first attempt of automating gyp file update https://github.com/nodejs/node/pull/47482, let's see how it goes and if its reliable we can add it to other dependencies, I'd  add a warn like:
`echo "Make sure to update the deps/uvwasi/uvwasi.gyp if any significant changes have occurred upstream"` in the bash script